feature/amazon/src/main/java/com/revenuecat/purchases/amazon/AmazonBilling.kt

```diff
@@ -107,6 +111,39 @@ internal class AmazonBilling constructor(
         )
     }
 
+    override fun normalizePurchaseData(
+        productID: String,
+        purchaseToken: String,
+        storeUserID: String,
+        onSuccess: (correctProductID: String) -> Unit,
+        onError: (PurchasesError) -> Unit
+    ) {
+        val currentlyCachedTokensToSkus = cache.getReceiptSkus()
+
+        currentlyCachedTokensToSkus[purchaseToken]?.let { sku ->
```

Comment:
do we have to worry about skus that were cached before this PR not being in the right format or anything?

Reply:
We are not changing the format, are we?

Reply:
didn't think so, was just confirming!